### PR TITLE
readme grammar

### PR DESCRIPTION
**Pull request by frosty00**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -178,13 +177,15 @@ The ccxt library currently supports the following 133 cryptocurrency exchange ma
...
 
-The list above is updated frequently, new crypto markets, altcoin exchanges, bug fixes, API endpoints are introduced and added on a regular basis. See the [Manual](https://github.com/ccxt/ccxt/wiki) for details. If you don't find a cryptocurrency exchange market in the list above and/or want another exchange to be added, post or send us a link to it by opening an issue here on GitHub or via email.
+The list above is updated frequently, new crypto markets, exchanges, bug fixes, and API endpoints are introduced on a regular basis. See the [Manual](https://github.com/ccxt/ccxt/wiki) for more details. If you can't find a cryptocurrency exchange in the list above and want it to be added, post a link to it by opening an issue here on GitHub or sending us an email.
```
Comment (kroitor) [inline README.md]: In this case, I think, it should end with:

> **post** a link to it by opening an issue here on GitHub **or sending** us an email.

↓

> **post** a link to it by opening an issue here on GitHub **or send** us an email.

?

Comment (xpl) [reply README.md]: Yeah, it changes the meaning... as they don't "post links" by "sending us an email", they _either_ "post links" or "send us emails".

**Comment by frosty00** [comment]
> ###Docker

**Comment by frosty00** [comment]
#4680

**Pull request merged**